### ISSUE TITLE
QA Ph4: Accessibility_Screen reader does not read all text on social tab of modal 

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#924]
With the NVDA screen reader enabled, the messages on the social tab of the modal are not read on mouse over. 

Steps: 
1. Enable the NVDA screen reader tool 
2. Go to https://io-webapp-staging.appspot.com/io2016/
3. Expand the live modal > go to the social tab 
4. Mouse over messages
5. Note issue

MS Edge Windows 10

![gio_924](https://cloud.githubusercontent.com/assets/7585604/15256882/79d67d7a-18f8-11e6-9b26-218a2b9ad3a3.PNG)
